render/expected/expected.go

```diff
@@ -17,18 +17,21 @@ var (
 		ID:                unknownPseudoNode1ID,
 		LabelMajor:        "10.10.10.10",
 		Pseudo:            true,
+		NodeMetadata:      report.MakeNodeMetadata(map[string]string{}),
```

Comment:
I think it you revert your change to trimNodeMetadata those errors will go away.